### PR TITLE
Setup maven central releases

### PR DESCRIPTION
**Pull request by BarkingBad**

### CODE REVIEW

**Comment by romanowski** [comment]
@tgodzik could you take a look?

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Release
+on:
+  push:
+    branches: [master, main]
```
Comment (tgodzik) [inline .github/workflows/release.yml]:
```suggestion
    branches: [main]
```
no need for master, especially that someone could push it at some point by mistake.

File: build.sbt [hunk 2]
```diff
@@ -1,11 +1,30 @@
...
+))
+
 val scala3Version = "3.0.0-RC2"
 
 lazy val root = project
   .in(file("."))
   .settings(
     organization := "com.virtuslab",
     name := "pretty-stacktraces",
-    version := "0.1.0",
```
Comment (tgodzik) [inline build.sbt:8]: You could also have `0.1.0-SNAPSHOT` or similar so it's easier to release locally

File: .github/workflows/release.yml (hunk 1 above)
Comment (BarkingBad) [reply .github/workflows/release.yml]: Fixed

File: build.sbt (hunk 2 above)
Comment (BarkingBad) [reply build.sbt:8]: Fixed

**Pull request merged**